packages/net/_test.pony

```diff
@@ -66,7 +66,7 @@ class _TestPing is UDPNotify
   =>
     _h.complete_action("ping receive")
 
-    let s = String .> append(consume data)
+    let s = recover val String.from_iso_array(consume data) end
```

Comment:
recover is a bit odd here since it's dropping from `iso^` to `val`, perhaps annotate `s: String val`